WordPressCom-Stats-iOS/InsightsTableViewController.m

```diff
@@ -291,6 +291,10 @@ - (void)retrieveStats
          [self setupRefreshControl];
          [self.refreshControl endRefreshing];
          
+         if (cachedDataWasUsed == NO) {
+             UIAccessibilityPostNotification(UIAccessibilityAnnouncementNotification, NSLocalizedString(@"Fetch of stats completed.", @"VoiceOver announcement of download finishing for stats."));
```

Comment:
I'm not the native English speaker here, but "Fetch of stats completed" sounds a bit too technical. How about something more like "Your stats have updated"